### PR TITLE
chore: remove references to `SlackMessage._channel_id`

### PR DESCRIPTION
**Pull request by joeyorlando**
# What this PR does

- As a follow-up to https://github.com/grafana/oncall/pull/5292, and now that `SlackMessage.channel` has been migrated via [`engine/apps/slack/migrations/0007_migrate_slackmessage_channel_id.py`](https://github.com/grafana/oncall/pull/5292/files#diff-8aebe133401715a4262baad9b2c5c9fc59367c18d6bd6ac2b3c462fcdabafd66), this PR removes reads/writes from `SlackMessage._channel_id` to `SlackMessage.channel`. In a separate PR I will focus on dropping that column from the model/db.
- Drops `SlackMessage.active_update_task_id`. There're zero references to this column in the codebase.
- Removes two Django `manage.py` commands that're no longer needed:
  - `engine/engine/management/commands/alertmanager_v2_migrate.py` (and it's associated tests)
  - `engine/engine/management/commands/batch_migrate_slack_message_channel.py`

## Checklist

- [x] Unit, integration, and e2e (if applicable) tests updated
- [x] Documentation added (or `pr:no public docs` PR label added if not required)
- [x] Added the relevant release notes label (see labels prefixed w/ `release:`). These labels dictate how your PR will
    show up in the autogenerated release notes.

### CODE REVIEW

File: engine/engine/management/commands/alertmanager_v2_migrate.py
Comment (joeyorlando) [inline engine/engine/management/commands/alertmanager_v2_migrate.py:1]: I don't believe this is needed anymore? (was removing `engine/engine/management/commands/batch_migrate_slack_message_channel.py` and decided to remove this as well)

File: engine/apps/alerts/models/alert_group.py [hunk 1]
```diff
@@ -417,6 +417,10 @@ def status(self) -> int:
     )
 
     prevent_posting_alerts = models.BooleanField(default=False)
+    """
+    TODO: this column is no longer used, drop it in a subsequent PR/release
```
Comment (joeyorlando) [inline engine/apps/alerts/models/alert_group.py:421]: hesitant to do this operation in this PR, clean-up for another day 🙂

File: engine/apps/slack/0009_remove_slackmessage_active_update_task_id_db.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+        ("slack", "0008_remove_slackmessage_active_update_task_id_state"),
+    ]
+
+    operations = [
+        common.migrations.remove_field.RemoveFieldDB(
+            model_name="SlackMessage",
+            name="active_update_task_id",
+            remove_state_migration=("slack", "0008_remove_slackmessage_active_update_task_id_state"),
+        ),
+    ]
```
Comment (joeyorlando) [inline engine/apps/slack/0009_remove_slackmessage_active_update_task_id_db.py:22]: this is instant (on a prod clone):
```bash
mysql> ALTER TABLE slack_slackmessage DROP COLUMN active_update_task_id;
Query OK, 0 rows affected (0.80 sec)
Records: 0  Duplicates: 0  Warnings: 0
```

**Review by vadimkerr — APPROVED** [review]
LGTM

**Pull request merged**